api/internal/handler/route/route.go

```diff
@@ -379,12 +380,25 @@ func (h *Handler) Create(c droplet.Context) (interface{}, error) {
 		}
 	}
 
-	ret, err := h.routeStore.Create(c.Context(), input)
+	// check name existed
+	ret, err := h.routeStore.List(c.Context(), store.ListInput{
```

Comment:
Can we use interface like `Get` so that the check can be faster.

Reply:
Can't, because the name is not index.

Reply:
We need to traverse the list to check if it exists.

Reply:
Got it.